app/services/git_based_domain_import_service.rb

```diff
@@ -62,6 +81,16 @@ def import(git_repo_id, branch_or_tag, tenant_id)
     domain.update_attribute(:enabled, true)
   end
 
+  def refresh(git_repo_id)
```

Comment:
Would prefer to pass the `git_repo` object here (as well as the other methods like `import` and `queue_refresh`).  Is there a specific reason to pass the ID?